From 2590c660518ac8e3ef2a56c82b6ecdb4c9a3f7de Mon Sep 17 00:00:00 2001
From: "Gupta Kumar, Sanjiv" <sanjiv.kumar.gupta@intel.com>
Date: Thu, 10 Aug 2017 16:40:08 +0530
Subject: [PATCH 1/3] Changes to build art-extension based on the
 ART_TO_VENDOR_ART_PATH

Change-Id: Ie443c0bd6b8f403ebf1170c0d796088f47024768
Category: Device Enablement
Domain: AOSP.ART-ME
Origin: Internal
Upstream-Candidate: no, proprietary
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-48718
Signed-off-by: Gupta Kumar, Sanjiv <sanjiv.kumar.gupta@intel.com>
---
 Android.bp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Android.bp b/Android.bp
index d0e22fb..6b12051 100644
--- a/Android.bp
+++ b/Android.bp
@@ -1,4 +1,8 @@
 // TODO: These should be handled with transitive static library dependencies
+
+//The subdir_prefix will be appended to all the subdirs. This way we can redirect to art-extension, based on the art-path.
+subdir_prefix = $(ART_TO_VENDOR_ART_PATH)
+
 art_static_dependencies = [
     // Note: the order is important because of static linking resolution.
     "libziparchive",
-- 
1.9.1

